<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="Player_Table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>621</width>
     <height>231</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Player ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Player Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Age</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gender</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Role</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="New_Player_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>New Player</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_to_Team_Button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to Team</string>
   </property>
  </widget>
  <widget class="QLabel" name="Error_Label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #f00;</string>
   </property>
   <property name="text">
    <string>Please specify Player</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
